project.properties

```diff
@@ -11,4 +11,4 @@
 split.density=false
 # Project target.
 target=android-10
-android.library.reference.1=../GreenDroid
+android.library.reference.1=../GreenDroid/GreenDroid
```

Comment:
Please move the Greendroid folder on your computer so it's accessible at `../GreenDroid` rather than modifying this variable.